web/src/main/webapp/WEB-INF/config-security/config-security.properties

```diff
@@ -104,3 +104,7 @@ geonetwork.https.url=@to_be_replaced_at_runtime@
 
 # The url to redirect to if a user logs in to one node then logs into another.
 wrongNodeRedirectURL=/@@nodeId@@/@@lang@@/node-change-warning?oldUserName=@@oldUserName@@&redirectedFrom=@@redirectedFrom@@&oldNodeId=@@oldNodeId@@
+
+# Password encrypter for passwords stored in database
+encrypter.algorithm=PBEWithMD5AndDES
+encrypter.password=default
```

Comment:
I would add those properties to main pom.xml properties in order to be able to build a webapp with a known encrypter values. If not, we will have to manually update the webapp to replace the "default" by the value generated. Having the config-sec.properties file outside the webapp would also help moving from one version to a new one.